### PR TITLE
Fix unclear documentation around wss connections.

### PR DESCRIPTION
**Pull request by michaelperret**
## What does this PR do?

Fix unclear documentation around wss connections.

## Checklist

- [X] All new functionality has tests.
- [X] All tests are passing.
- [X] New or changed API methods have been documented.

/cc @hph

### CODE REVIEW

**Comment by leesio** [comment]
Thanks for the PR!

I wonder whether you could explain a bit more about the issue you had that this documentation makes clear? I'm a little confused about it.

**Comment by michaelperret** [comment]
Hey @leesio sorry about the delay, just seeing this. Basically I had implemented it as follows:

```
const socket = new Pusher(APP_KEY, {
  cluster: APP_CLUSTER,
  enabledTransports: ['wss']
});
```

under the assumption that by specifying `wss` the `encrypted=True` was implied.

**Comment by leesio** [comment]
Time for me to apologise for the delay. I completely missed this. I'm going to be spending a bit of time with pusher-js today so I'll try to check this behaviour out and update the docs accordingly.

Thanks for the PR

File: README.md [hunk 1]
```diff
@@ -223,6 +223,17 @@ const socket = new Pusher(APP_KEY, {
...
 ```
 
+Note: if you intend to use secure websockets, or `wss`, you can not simply specify `wss` in `enabledTransports`, you must specify `ws` in `enabledTransports` as well as set the `encrypted` option to `true`.
+
+```js
+// Only use secure WebSockets
+const socket = new Pusher(APP_KEY, {
+  cluster: APP_CLUSTER,
+  enabledTransports: ['ws'],
+  encrypted: true
```
Comment (leesio) [inline README.md]:
```suggestion
  forceTLS: true
```

**Comment by leesio** [comment]
@michaelperret I lost track of this PR, I added a new commit in #390 and I'm merging it now.

Thanks very much for your contribution.

**Pull request closed without merging**